apps/condo/domains/marketplace/hooks/useMarketplaceServicesTableColumns.tsx

```diff
@@ -0,0 +1,152 @@
+    return useMemo(() => {
+        return [
+            {
+                title: SkuTitle,
+                sortOrder: get(sorterMap, 'sku'),
+                filteredValue: getFilteredValue(filters, 'sku'),
+                key: 'sku',
+                dataIndex: 'sku',
+                sorter: true,
+                width: '8%',
+                render: render,
+                filterDropdown: getFilterDropdownByKey(filterMetas, 'sku'),
+            },
+            {
+                title: SkuNameTitle,
+                filteredValue: getFilteredValue(filters, 'name'),
+                key: 'name',
+                dataIndex: 'name',
+                width: '23%',
+                filterDropdown: getFilterDropdownByKey(filterMetas, 'name'),
+                render: render,
+                filterIcon: getFilterIcon,
+            },
+            {
+                title: CategoryTitle,
+                filteredValue: getFilteredValue(filters, 'category'),
+                key: 'category',
+                dataIndex: 'marketCategory',
+                width: '23%',
+                filterDropdown: getFilterDropdownByKey(filterMetas, 'category'),
+                render: (category) => {
+                    const categoryName = intl.formatMessage( { id: `marketplace.marketCategory.${category.name}.name` })
+
+                    if (category.parentCategory) {
+                        const parentCategoryName = intl.formatMessage( { id: `marketplace.marketCategory.${category.parentCategory.name}.name` })
+
+                        if (subcategoryCounterGropedByCategoryId[category.parentCategory.id] > 1) return render(`${parentCategoryName}»${categoryName}`)
+                        return render(parentCategoryName)
+                    }
+                    return render(categoryName)
+                },
+                filterIcon: getFilterIcon,
+            },
+            {
+                title: ScopeTitle,
+                key: 'scope',
+                width: '25%',
+                render: (marketItem) => {
+                    const componentsToRender = []
+                    const priceForAllProperties = get(processedScopes, [marketItem.id, 'priceForAllProperties'])
+                    if (priceForAllProperties) componentsToRender.push(<div key='priceForAllProperties'>
+                        {getMoneyRender(intl, get(priceForAllProperties, 'currency', 'RUB'))(get(priceForAllProperties, 'price'))}
+                        <Typography.Text type='secondary' style={{ margin: '10px' }}>({AllPropertiesMessage})</Typography.Text>
+                    </div>)
+
+                    for (const price in processedScopes[marketItem.id]) {
+                        const items = processedScopes[marketItem.id][price]
+                        const address = get(items[0], 'address')
+
+                        componentsToRender.push(<div key={address}>
+                            {getMoneyRender(intl, get(items[0], 'currency', 'RUB'))(price)}
```

Comment:
You can use `getMoneyRender` from `'@condo/domains/marketplace/utils/clientSchema/Invoice'` to handle case with `isMin` price.
Also there can be Contract price (`price` = 0 and `isMin` = true)